service/routes.go

```diff
@@ -0,0 +1,21 @@
+package service
+
+import (
+	"github.com/gorilla/mux"
+	"github.com/rancher/go-rancher/api"
+	"github.com/rancher/go-rancher/client"
+	//This should be v2, supporting schemas
+	"net/http"
+)
+
+var schemas *client.Schemas
+var router *mux.Router
+
+func NewRouter() *mux.Router {
+	schemas = &client.Schemas{}
+	router = mux.NewRouter().StrictSlash(true)
+	router.Methods("POST").Path("/v1-webhooks-generate").Handler(api.ApiHandler(schemas, http.HandlerFunc(ConstructPayload)))
+	router.Methods("POST").Path("/v1-webhooks-execute").Handler(api.ApiHandler(schemas, http.HandlerFunc(Execute)))
+	router.Methods("GET").Path("/v1-webhooks-schemas").Handler(api.ApiHandler(schemas, http.HandlerFunc(DisplaySchemas)))
```

Comment:
schemas endpoint should be /v1-webhooks-generate/schemas. We won't need schemas for execute. Also, instead of execute, use the verb receiver instead execute. Darren suggested it and I like it better.